CI/env.sh

```diff
@@ -20,7 +20,7 @@ export  FC=`which gfortran`
 if [ "$COMPILER" == "gcc" ]; then
   export  CC=`which gcc`
   export CXX=`which g++`
-  export jobs="4"
+  export ci_jobs="4"
```

Comment:
Doesn't this line overwrite the previous declaration of `ci_jobs`?

Reply:
It does, but it's on purpose.  On our CI machines, gcc manages to consume all the memory and the job crashes/fails.  We need to limit to 4 CPUs

Reply:
That makes sense